### ISSUE TITLE
Process Explorer: window controls are not accessible by keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51752]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Keyboard,MAS 2.1.1,MACOS High Sierra 10.13.5 Beta (17F70a)

**Repro Steps:**
1. Launch VS Code.
2.Navigate to Help(Alt+H) menu option and select "Open Process Explorer" from dropdown.
3. Check the keyboard navigation.

**Actual:**
Minimize,maximize and close button is not accessible by keyboard.

**Expected:**
Minimize,maximize and close button should accessible by keyboard.
Same issue observed in MAC also.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to access the controls.
**Attachment Reference:**

**MAS Reference:**
MAS 2.1.1
[[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}]](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment Reference:**
![accessibility a11y_vscode_processexplorer_keyboard](https://user-images.githubusercontent.com/39204909/41393585-231b0be8-6fc4-11e8-9bbd-8380aaebcfac.png)

[[Accessibility]A11y_VSCode_ProcessExplorer_Keyboard1.pptx](https://github.com/Microsoft/vscode/files/2101685/Accessibility.A11y_VSCode_ProcessExplorer_Keyboard1.pptx)

**Comment by RMacfarlane** [comment]
@AccessibilityTestingTeam-TCS What is the expected behavior for these? The menu bar is native from the OS, so on Windows, pressing `Alt+Spacebar` shows a context menu, and the keybindings `Ctrl+Up arrow`, `Ctrl+Down arrow` and `Alt+F4` work for minimizing, maximizing, and closing.

**Comment by isidorn** [comment]
@RMacfarlane that sounds to me like they are accessible via keyboard. Not sure what is the problem here, not everything should be possible to tab into. 
You can say the same for our main window.

**Comment by AccessibilityTestingTeam-TCS** [comment]
It is working fine with  shortcuts given above for minimizing, maximizing, and closing.

**Comment by isidorn** [comment]
Then there seems to be nothing to be done here -> closing

**Comment by AccessibilityTestingTeam-TCS** [comment]
It is working fine,hence closing this bug.
